app/controllers/Login.scala

```diff
@@ -34,7 +33,7 @@ class Login(
    * Looks up user's identity via Google
    */
   def oauth2Callback: Action[AnyContent] = Action.async { implicit request =>
-    processOauth2Callback(requiredGoogleGroups, googleGroupChecker)
+    processOauth2Callback()
```

Comment:
this means we no longer check group membership during the initial google auth step, and we instead do it on every request